### DISCUSSION TITLE
Redirecting response from GRPC request to a file

### DISCUSSION THREAD

**Discussion by Steve4765** [AnWeber/httpyac#296] [General]
How do we redirect the response from a GRPC request to a file ?
The example 

```
GET  https://httpbin.org/anything HTTP/1.1

>> ./outputRedirection.json
```

works.

But it doesn't seem to be working for .http files that make GRPC requests

**Comment by AnWeber** [comment]
@Steve4765 Unfortunately, this is a bug (#297) right now. I forgot to fill the rawbody in the GRPC response. But this is needed for the output redirection. With the next version it works. A workaround would currently be to simply save the body itself using NodeJS script and `fs`.

**Reply by Steve4765** [reply]
Thanks for fixing this bug. I'll wait for the next version.

**Comment by AnWeber** [comment]
I am creating this right now. in 20 minutes
